exchanges/stream/stream_types.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
```

Comment:
A somewhat optional request is perhaps this is a good point to add a `readme.md` to the stream folder

Reply:
Happy to do it, what would you like in there?

Reply:
Added a basic [something](https://github.com/thrasher-corp/gocryptotrader/pull/1580/commits/818584fbb4d65c78eb442ab8043ee41405e94928)

Add what more you want when you can.